File: FitTrackApp/app/src/main/java/com/example/fittrackapp/models/WeeklyPlan.java
package com.example.fittrackapp.models;

public class WeeklyPlan {
    public int planId;
    public int userId;
    public String dayOfWeek;
    public int exerciseId;

    public WeeklyPlan(int userId, String dayOfWeek, int exerciseId) {
        this.userId = userId;
        this.dayOfWeek = dayOfWeek;
        this.exerciseId = exerciseId;
    }
}

File: backend/models/WeeklyPlan.cs
using System.ComponentModel.DataAnnotations;

namespace FitTrackBackend.Models;

public class WeeklyPlan
{
    [Key]
    public int PlanId { get; set; }

    public int UserId { get; set; }

    public string DayOfWeek { get; set; } = "";

    public int ExerciseId { get; set; }

    public DateTime CreatedAt { get; set; } = DateTime.UtcNow;
}